example/lambdas/python-reference-task/package.json

```diff
@@ -11,11 +11,12 @@
   },
   "scripts": {
     "test": "true",
-    "python-lint": "pylint *.py",
+    "python-lint": "pipenv run pylint *.py",
     "lint": "npm run python-lint",
-    "clean": "rm -rf dist && rm -rf lib && mkdir dist && mkdir lib",
-    "build": "npm run clean && pip install -r requirements-dev.txt && pip install -r requirements.txt -t ./dist && cp *.py ./dist/ && cd ./dist && find . -type f | egrep -v '\\.json$' | xargs node ../../../../bin/zip.js lambda.zip && cd ..",
-    "package": "npm run build",
+    "clean": "rm -rf dist && rm -rf lib && rm -rf .venv && mkdir dist && mkdir lib",
+    "build": "pip install pipenv && PIPENV_VENV_IN_PROJECT=1 pipenv install --dev --deploy --ignore-pipfile",
+    "prepare": "npm run build",
+    "package": "npm run clean && ../../scripts/package-pipenv-deps.sh $(pwd) && cp *.py ./dist/ && cd ./dist && find . -type f -print0 | xargs -0 node ../../../../bin/zip.js lambda.zip && cd ..",
```

Comment:
Given the complexity of this npm script and the fact that currently we only need this script here, I'm somewhat inclined to move it into this folder and called it `package.sh` or something. And then move all the necessary commands in this NPM script into the bash script

It seems like we could also then just call `npm run build` in the bash script instead of repeating the `pipenv install` commands in the packaging script

Reply:
https://github.com/nasa/cumulus/pull/2038/commits/a167671b8514c0994b45218b1acc767bae029fe0